src/storage-client/src/controller.rs

```diff
@@ -1010,6 +1023,27 @@ where
                             self.truncate_managed_collection(id).await;
                             self.initialize_shard_mapping().await;
                         }
+                        IntrospectionType::StorageReplicaMetrics => {
+                            // The metrics scraper assumes that the collection
+                            // sums up to empty when starting up. When needed,
```

Comment:
as in, starting up a new source/sink? Or on every restart?

Reply:
When the controller is starting up. We currently truncate all managed collections when that happens, see block right above.